docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -40,7 +53,8 @@ It's a good idea to mark it down, but should you forget, you can always have a l
 
```

Comment:
Shouldn't the line above this one say `Total results: 4` instead of 1?

Also, perhaps we should make a note that in some places (such as in this table), `PK` may also be referred to as `Id`.

Reply:
Huh, I actually didn't notice the `Id` one. It should say PK, no? That seems like it just hasn't been fixed yet.

Reply:
I added a note, even though it's a little strange that the column header isn't `PK`.